src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -0,0 +1,76 @@
+cfg_if! {
+    if #[cfg(target_arch = "x86_64")] {
+        mod x86_64;
+        pub use self::x86_64::*;
+    } else {
+        // ...
+    }
+}
```

Comment:
Could you put this at the end of the file? Stylistically that tends to be its current location